frontend/src/pages/ExplorePage.js

```diff
@@ -1,10 +1,14 @@
 import styled from "styled-components/macro";
 import StreetMap from "../components/StreetMap";
 
-export default function ExplorePage({ points, setIsLoading }) {
+export default function ExplorePage({ points, setIsLoading, setPoints }) {
```

Comment:
When passing own all the props you can use the spread operator like so:
```javascript
export default function ExplorePage(props) {
  return (
    <Wrapper>
      <StreetMap {...props} />
    </Wrapper>
  );
```

Reply:
That's nifty, thanks!